Document the new plan change and the incompatible features feature (#967)
dnsimple/dnsimple-support: content/articles/changing-plans.markdown

As a DNSimple customer, you might require features that aren't available on your current plan. This article discusses how to switch to the most appropriate subscription to meet your current needs.

> There are two different kind of change plans based on how they are going to be billed.

Reviewer: ```suggestion
There are two different types of plan changes based on how they are going to be billed.
```

## Pre-Renewal change plans

> These kind of changes are billed immediately and updates the [billing cycle](/articles/yearly-billing).

Reviewer: ```suggestion
These plan changes are billed immediately and update the [billing cycle](/articles/yearly-billing).
```

> In case there are remaining days in your current subscription plan, you will receive a prorated discount. This discount doesn't consider extras, like email forwards or extra zones, which you'll be billed for again as part of the plan change. Read more about [subscription proration](/articles/understanding-invoice).

Reviewer: ```suggestion
You will receive credit for all the unused time of your current subscription plan. This credit doesn't include extra items such as email forwards or extra zones. These will be billed again as part of the plan change. Read more about [subscription proration](/articles/understanding-invoice).
```

## On-Renewal change plans

These kind of changes are billed without changing the billing cyle. They will be billed along with the renewal of the plan, adhering to the existing billing cycle.

> You can cancel a scheduled plan change in your account settings:

Reviewer: ```suggestion
You can cancel these plan changes in your account settings:
```
